requirements/gateway-requirements.txt

```diff
@@ -5,3 +5,4 @@
 pydantic<3,>=1.0
 fastapi<1
 uvicorn<1
+httpx<1
```

Comment:
Is `httpx` a test-only dependency?

Reply:
ah... just validated that it's a dependency for the fastapi.testclient ! 
Moving this import to test-requirements.txt